scripts/kubewarden-load-policies.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -euo pipefail
+
+policies="kubewarden-policies.tar.gz"
+list="kubewarden-policies.txt"
+
+usage () {
+    echo "USAGE: $0 [--policies kubewarden-policies.tar.gz] --registry my.registry.com:5000"
+    echo "  [-l|--policies-list path] text file with list of policies; one image per line."
+    echo "  [-p|--policies path] tar.gz generated by kwctl save."
+    echo "  [-r|--registry registry:port] target private registry:port."
+    echo "  [-s|--sources-path path] kwctl sources path."
+    echo "  [-h|--help] Usage message"
+}
+
+pushPolicy() {
+    newPolicyUrl=$1
+    if [[ -n $sourcesPath ]]; then
+        kwctl push "$policy" "$newPolicyUrl" --sources-path "$sourcesPath"
+    else
+        kwctl push "$policy" "$newPolicyUrl"
+    fi
+}
+
+while [[ $# -gt 0 ]]; do
+    key="$1"
+    case $key in
+        -r|--registry)
+        registry="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -l|--policies-list)
+        list="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -p|--policies)
+        policies="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -s|--sources-path)
+        sourcesPath="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -h|--help)
+        help="true"
+        shift
+        ;;
+        *)
+        usage
+        exit 1
+        ;;
+    esac
+done
+if [[ -v help ]]; then
+    usage
+    exit 0
+fi
+if [[ -z $registry ]]; then
```

Comment:
Prevents ugly error if variable is not set
```suggestion
if [[ -z ${registry:-} ]]; then
```